.github/workflows/build.yaml

```
@@ -0,0 +1,190 @@
+
+env:
+  DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      # Using matrix for potential future expansion. For now, it runs a single service.
+      matrix:
+        service: [ "${{ github.event.inputs.service }}" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)